BiweeklyMeetings/2022-07-05.md

```diff
@@ -0,0 +1,274 @@
+
+**New CIPs Review**: CIPs up for review should be looked over collectively, with discussion where needed. (on top of the asynchronous reviews)  
+
+PR -> ‘Draft’: Needs format + approval.  
+‘Draft’ -> ‘Proposed’: Needs a PLAN towards Active + implementation.  
+‘Proposed’ -> ‘Active’:  Objective criteria as laid out observed, and consensus agreeing.   
+**Current Discussions**: What the current CIPs discussions are on social media / forums / Discord.  
+
+**Issues**: Review of the open Issues on the CIP GitHub 
+
+**Close**: Recap of actions taken and decisions. List the CIPs that are due for review.  
+
+
+## July 05 2022 Transcript
+**Attending Editors**: Matthias Benkort, Sebastien Guillemot, ~Frederic Johnson~, ~Robert Phair~. 
+**Guests**:
+
+### Triage
+
+#### PR274
+[PR274 CIP-0059? | Terminology Surrounding Core Features](https://github.com/cardano-foundation/CIPs/pull/274)
+
+**Matthias** - :  Yeah, we can start already and go right in the first item, right, CIP 59, which was proposed by Jared, I think after a chat on a different CIP or different debates around Vasil versus Daedalus, and what do names mean exactly in Cardano. A lot of people seem to have gotten confused by the names being employed. So he wrote a small, informational CIP, there is not really much to debate in a way except maybe if people disagree with... I know the similar thing given to that and they want to change it, but since there is a type history already. It just explains where the name comes from, what they are and the rationale behind is why they were people like that. Like between era, and a phase, and a network and protocol. And he has also put up a nice table which summarizes all of that, which is nice, so it gives just a bit more clarity about naming.
+
+**Matthias** - :  And yeah, so I pointed to him last time a few things to do regarding the proposal of just the editorial stuff, putting a number, putting the readme in the correct folder. He did all of that and he already marked it as active as I suggested, just because as I said, it's an informational CIP explaining the terminology picked by the core team. It's from the core team itself and it's also recording or explaining things from the past six eras or so. So yeah, there is no point going to things like propose, and then active, and then whatever, unless people agree with that. Okay. 
+
+**Sebastien** -, and your opinion maybe, if you are here.
+
+**Sebastien** -:  I joined probably through the conversation, so I'm missing the beginning.
+
+**Matthias** - :  Yeah. Okay, sorry. We are talking about CIP 59, the terminologies surrounding co-features, which was proposed by Jared some time ago to clarify a bit of the terminology around the co-features. And I was saying, this one could probably just go as active right away because it's just explaining what's in there.
```

Comment:
```suggestion
**Matthias** - :  Yeah. Okay, sorry. We are talking about CIP 59, the terminologies surrounding core features, which was proposed by Jared some time ago to clarify a bit of the terminology around the core features. And I was saying, this one could probably just go as active right away because it's just explaining what's in there.
```